db/seeds/development/driver_commission_history.seeds.rb

```diff
@@ -0,0 +1,23 @@
+Octopus.using(:grossman) do
+  10.times do
+    DriverCommissionHistory.create(
+            :driver_id => Faker::Number.number(4),
+            :backhauls => 'BACKHAULS',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.